Makefile

```diff
@@ -7,6 +7,8 @@ PODMAN ?= podman
 
 BUILDTAGS ?= ""
 BUILDER ?= "docker"
+EXTRA_BUILD_ARGS ?= $(null)
```

Comment:
what's the difference between $(null) and "" in this context? Can we use either $(null) or "" for all variables?

Reply:
I did not dig deeper but if I had "", docker build fails

Reply:
My guess this is because you didn't use "" around variable reference in build-mages.sh:
```if [ -n ${EXTRA_BUILD_ARGS} ]; then```

Reply:
I'll try.